.github/workflows/ci.yml

```diff
@@ -113,16 +113,79 @@ jobs:
             -   name: Checkout
                 uses: actions/checkout@v3
 
+            -   name: Setup PHP to install dependencies
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 5.5
+                    ini-values: error_reporting=-1, display_errors=On, zend.assertions=1
+                    coverage: none
+                    # Autoload files generated with Composer 2.3+ are not compatible with PHP < 7.0.
+                    tools: composer:2.2
+
+            -   name: Install Composer dependencies
+                uses: ramsey/composer-install@v2
+                with:
+                    composer-options: "--no-dev --optimize-autoloader"
+
+            -   name: Setup PHP to generate the PHAR
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: 8.1
+                    extensions: exif, phar, openssl, sodium
+                    coverage: none
+                    ini-values: phar.readonly=Off, error_reporting=-1, display_errors=On, zend.assertions=1
+                    # Autoload files generated with Composer 2.3+ are not compatible with PHP < 7.0.
+                    tools: composer:2.2
+
+            # Note: do NOT turn on the requirement checker in the box config as it is no longer
+            # compatible with PHP < 7.2.
+            -   name: Install Box
+                run: wget https://github.com/humbug/box/releases/latest/download/box.phar -O box.phar && chmod 0755 box.phar && pwd
+
+            -   name: Validate configuration
+                run: php box.phar validate -i box.json
+
+            -   name: Building binary...
+                run: php box.phar compile -v --config=box.json
+
+            -   name: Show info about the build phar
+                run: php box.phar info -l ./build/artifacts/php-coveralls.phar
+
+            -   name: Upload the Phar artifact
+                uses: actions/upload-artifact@v3
+                with:
+                    name: php-coveralls-phar
```

Comment:
```suggestion
                    name: php-coveralls.phar
```
perhaps?

Reply:
This is not a file name, but the canonical name for the artifact.
See: https://github.com/actions/upload-artifact